packages/lodestar/src/chain/factory/block/index.ts

```diff
@@ -22,12 +22,12 @@ export async function assembleBlock(
   eth1: IEth1Notifier,
   slot: Slot,
   randao: bytes96
-): Promise<BeaconBlock|null> {
+): Promise<BeaconBlock | null> {
   const [parentBlock, currentState] = await Promise.all([
     db.block.get(chain.forkChoice.head()),
-    db.state.getLatest(),
+    db.state.get(chain.forkChoice.head()),
```

Comment:
I think what @mpetrunic means is:
+ Get `parentBlock: BeaconBlock` like above
+ Get `stateRoot` from `parentBlock.BeaconBlock`
+ Get state like `db.state.get(stateRoot)`

So instead of doing Promise.all we get block first and then state.